ingest/defaults/config.yaml

```diff
@@ -66,7 +63,7 @@ curate:
   # Currently accepts any characters because we do not have a clear standard for strain names across pathogens
   strain_regex: "^.+$"
   # Back up strain name field to use if "strain" doesn"t match regex above
-  strain_backup_fields: ["accession"]
+  strain_backup_fields: []
```

Comment:
[non-blocking]

I suppose there was a reason for deleting the "accession" as a backup field for strain name?

I see that the inferred command

```bash
| ./vendored/transform-strain-names   \
 --strain-regex ^.+$  \
 --backup-fields  \
 | augur curate format-dates \
 ...
 ```
 
still runs without backup fields, so I don't have a strong opinion on keeping or dropping.

Reply:
Nevermind, I see the reason in the commit message: https://github.com/nextstrain/rabies/pull/2/commits/618269bd8ae46183d51c319060a09707d33619e1

> We plan to use accession as the strain ID field in the phylogenetic workflow, and so we will not need to replace empty strain names.